ironfish-cli/src/commands/chain/download.ts

```diff
@@ -44,11 +27,11 @@ export default class Download extends IronfishCommand {
       required: false,
       description: 'Path to a downloaded snapshot file to import',
     }),
-    outputPath: Flags.string({
+    output: Flags.string({
```

Comment:
Even though this likely isn't used often, we should add a note to the release notes that this is a breaking change